carbon-design-system/carbon
# Added compliance footer to v11

Closes #15643

#### Changelog

**New**

- Added footer to storybook

#### Testing / Reviewing

- Open storybook to check the design
- Check if the place that was placed the footer in the code make sense
